backend/core/Extensions/QueryableExtensions.cs

```diff
@@ -122,7 +122,7 @@ private static Type GetPropertyPathType<T>(string path)
             foreach (var part in path.Split('.'))
             {
                 var prop = type.GetCachedProperties().FirstOrDefault(p => p.Name == part) ?? throw new ArgumentException($"Property path '{type.Name}.{path}' is invalid.", nameof(path));
-                type = prop.PropertyType.IsEnumerable() ? prop.PropertyType.GetItemType() : prop.PropertyType;
+                type = prop.PropertyType.IsEnumerable() && prop.PropertyType != typeof(string) ? prop.PropertyType.GetItemType() : prop.PropertyType;
```

Comment:
Hopefully resolve bug that shows up in OpenShift but not locally for some reason.